### PR TITLE
Support --input flag to read targets from file

### PR DESCRIPTION
**Pull request by mihails-strasuns-sociomantic**
Trying to pass unknown amount of fully qualified paths as CLI
arguments may hit shell char length limit eventually. Using file
to temporarily store the list is more reliable.

Will be used in updated makd integration for `make d2conv`

### CODE REVIEW

**Comment by mihails-strasuns-sociomantic** [comment]
I have split this change from dsymbol PR as it is completely independent.

File: src/d1to2fix/main.d [hunk 1]
```diff
@@ -102,14 +102,32 @@ int main ( string[] args )
...
+    if (args[1][0] == '@')
+    {
+        fileNames = readText(args[1][1 .. $])
+            .splitter("\n")
+            .filter!(s => s.length)
+            .array();
+    }
+    else
+        fileNames = args[1 .. $];
+
```
Comment (leandro-lucarella-sociomantic) [inline src/d1to2fix/main.d:155]: Mmm, so is either `d1to2fix @file` or `d1to2fix file1 ... fileN` but `d1to2fix @files1 file1` can't be used? In that case (or in any case) why not using a command line option like -f (or even stdin) to get multiple files? I always found `@file` super weird...

**Comment by mihails-strasuns-sociomantic** [comment]
Updated to use dedicated `--input` flag

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,3 @@
 build/
 deb/*.deb
-.converted
+tests/*.converted
```
Comment (leandro-lucarella-sociomantic) [inline .gitignore:3]: Mmm, did this location really changed in this commit or it was an unrelated "bug fix"?

Comment (mihails-strasuns-sociomantic) [reply .gitignore:3]: Bug fix, I'll split

**Comment by mihails-strasuns-sociomantic** [comment]
Anything else?

**Pull request merged**